apps/src/applab/designElements/library.js

```diff
@@ -160,6 +160,12 @@ export default {
     }
     // Unknown elements are expected. Return null because we don't know type.
     if (allowUnknown) {
+      console.warn(
```

Comment:
Synced offline: Don't need to log to firehose. The intent of having this warning is to help with diagnosing issues that come in via Zendesk. Issues like this will persist, so we won't need to track a single event in time. Additionally, it's not particularly valuable to track how many times this happens.